plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -31,8 +31,18 @@ import {
 } from '@superset-ui/chart-controls';
 import { validateNonEmpty } from '@superset-ui/validator';
 import { smartDateFormatter } from '@superset-ui/time-format';
+import i18n from './i18n';
 
-export const PAGE_SIZE_OPTIONS = formatSelectOptions<number>([[0, t('All')], 10, 20, 50, 100, 200]);
+addLocaleData(i18n);
```

Comment:
Had to inject the locale data here because it was used by the `controlPanel` module before it can be imported. Ideally this should be in a more stable place, e.g., `Plugin.setup(...)` or `Plugin.contructor(...)`, but that requires refactoring `controlPanel` to be loaded async. Let's leave it to a future PR.